### PR TITLE
Add SmallRye Config 1.9 Blog Post.

### PR DESCRIPTION
**Pull request by radcortez**

### CODE REVIEW

File: _posts/2020-09-24-smallrye-config-1-9.adoc [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+[source,java]
+----
+@ConfigMapping(prefix = "server")
+interface Server {
+    String host();
+
+    int port();
+}
+----
```
Comment (Ladicek) [inline _posts/2020-09-24-smallrye-config-1-9.adoc:25]: Isn't something like this present in MP Config 2.0? Perhaps mention that?

File: _posts/2020-09-24-smallrye-config-1-9.adoc [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+The `Server` interface retrieves and maps configuration properties with the name `server.host` into the
+`Server.host()` and `server.port` into `Server.port()` method. The configuration property name to lookup
+builds from the `@ConfigMapping` annotation prefix and the method name with `,` (dot) as the separator. The
```
Comment (Ladicek) [inline _posts/2020-09-24-smallrye-config-1-9.adoc:29]: You say "dot", but write a comma :-)

**Comment by Ladicek** [comment]
LGTM otherwise :-)

File: _posts/2020-09-24-smallrye-config-1-9.adoc (hunk 1 above)
Comment (radcortez) [reply _posts/2020-09-24-smallrye-config-1-9.adoc:25]: Sure!

File: _posts/2020-09-24-smallrye-config-1-9.adoc (hunk 2 above)
Comment (radcortez) [reply _posts/2020-09-24-smallrye-config-1-9.adoc:29]: Ops :)

**Comment by radcortez** [comment]
> LGTM otherwise :-)

Thanks for the review :)

**Pull request merged**